homework/hoanghai/lesson7-2/513-FindBottomLeftTreeValue.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Definition for a binary tree node.
+ * function TreeNode(val, left, right) {
+ *     this.val = (val===undefined ? 0 : val)
+ *     this.left = (left===undefined ? null : left)
+ *     this.right = (right===undefined ? null : right)
+ * }
+ */
+/**
+ * @param {TreeNode} root
+ * @return {number}
+ */
+ var findBottomLeftValue = function(root) {
+    let queue = [root]
+    let answer = []
+    let previous = []
+    while(queue.length) {
+        // xet lan luot tung tang, boi vi answer la ket qua ben trai cua tang duoi cung,ma ben trai luon duoc push vao truoc nen previous 0 la ket qua can tim
```

Comment:
better use English for everything. It helps your brain think in English => eliminate the "translate" part when you explain answer to interviewer.